Cargo.toml

```diff
@@ -44,7 +44,7 @@ release_max_level_info  = []
 release_max_level_debug = []
 release_max_level_trace = []
 
-std = []
+std = ["value-bag?/std"]
```

Comment:
Is this supported in Rust 1.60 (our MSRV)?

Reply:
https://doc.rust-lang.org/cargo/reference/features.html

According to the doc linked, it's available starting from 1.60 so it should be fine.